test/test_with_xpress.jl

```diff
@@ -400,6 +400,44 @@ check to make sure that PV does NOT export unless the site load is met first for
               if results["PV"]["year_one_to_grid_series_kw"][i] > 0)
 end
 
+@testset "Battery degradation replacement strategy" begin
```

Comment:
@zolanaj I added this test to test the outputs as well as model expression values. Since solve time is not an issue I am sticking with a simple LargeOffice load profile.

I had to change the result JSON tests to inequalities because we get different results on github action than those when testing this scenario by itself.

Reply:
@rathod-b some values like the SOH indicator at the end are going to be stable while a lot of these others will change with solver settings.  We can construct a more unit-test-like load profile to incentivize a certain behavior and make the solution more stable (something like a flat load with hourly cost that oscillates from very expensive to free every other period.)